### PR TITLE
PHEE-386,373 Update circle CI config of with integration test and host charts

### PR DESCRIPTION
**Pull request by dhruvsonagara**
## Description

[PHEE-386 Update circle CI config of micro services with integration test](https://fynarfin.atlassian.net/browse/PHEE-386?atlOrigin=eyJpIjoiZjczZjdlOWI5MTA1NDgyN2I3OTQ4M2NiMzk5ZTVkMGYiLCJwIjoiamlyYS1zbGFjay1pbnQifQ)
[PHEE-373 build & host ph-ee-engine chart, g2p-sandbox chart and G2P sand…](https://fynarfin.atlassian.net/browse/PHEE-373?atlOrigin=eyJpIjoiYmVjYTRkYWEyNmMxNGIyNWI2ODk5OTI3OGMxMTdjODYiLCJwIjoiamlyYS1zbGFjay1pbnQifQ)

## Checklist

Please make sure these boxes are checked before submitting your pull request - thanks!

- [ ] Design related bullet points or design document link related to this PR added in the description above.

- [ ] Updated corresponding Postman Collection or Api documentation for the changes in this PR.

- [ ] Create/update unit or integration tests for verifying the changes made.

- [ ] Add required Swagger annotation and update API documentation with details of any API changes if applicable

- [ ] Followed the naming conventions as given in https://docs.google.com/document/d/1Q4vaMSzrTxxh9TS0RILuNkSkYCxotuYk1Xe0CMIkkCU/edit?usp=sharing

### CODE REVIEW

**Comment by arkadasfynarfin** [comment]
@dhruvsonagara Looks good to me.
CC: @fynmanoj

File: .circleci/config.yml [hunk 1]
```diff
@@ -32,7 +38,7 @@ jobs:
             fi
 
             # Build and tag the Docker image
-            ./gradlew build
+            ./gradlew bootJar
```
Comment (fynmanoj) [inline .circleci/config.yml:41]: Don't remove build , make sure that checkstyles are executed

File: .circleci/config.yml [hunk 2]
```diff
@@ -70,6 +80,120 @@ jobs:
           name: Push Docker image to DockerHub
           command: docker push openmf/ph-ee-connector-channel:latest
 
+      - run: |
+          kubectl config use-context arn:aws:eks:$REGION:$ACCOUNT_ID:cluster/sit
```
Comment (fynmanoj) [inline .circleci/config.yml:84]: For bulk, ams mifos, channel and ops app, the restart logic is already added. You can skip this

File: .circleci/config.yml [hunk 3]
```diff
@@ -70,6 +80,120 @@ jobs:
...
+
+  build-and-host-g2p-sandbox:
+    docker:
+      - image: cimg/python:3.10
+    working_directory: ~/repo
+    environment:
+      TERM: dumb
+    steps:
+      - run: git clone https://github.com/openMF/ph-ee-env-template
+      - run: rm -f helm/g2p-sandbox/Chart.lock helm/g2p-sandbox/requirements.lock helm/g2p-sandbox/charts/*
```
Comment (fynmanoj) [inline .circleci/config.yml:120]: Here you have to give proper path. Else it won't work with got clone

Comment (fynmanoj) [reply .circleci/config.yml:120]: Suggestion: use ph-ee-env-template in the path

**Review by fynmanoj — CHANGES_REQUESTED** [review]
Can you check circle ci orn creation and may be try to avoid these code duplications?

**Review by fynmanoj — APPROVED** [review]

**Pull request merged**